Header header

# True if battery is low
bool battery_low
bool is_charging
bool is_fully_charged
